src/components/existing-request/existing-request-display.vue

```diff
@@ -417,8 +412,10 @@ export default class ExistingRequestDisplay extends Mixins(
    */
   get showIncorporateButton (): boolean {
     return (
-      this.isBenefitCompany(this.nr) &&
-      this.actions.includes(NrAction.INCORPORATE)
+      this.isSupportedEntity(this.nr) &&
+      this.nr.request_action_cd &&
+      this.nr.request_action_cd === NrRequestActionCodes.NEW_BUSINESS &&
+      NrState.APPROVED === this.nr.state
```

Comment:
We'll be showing the button if the current entity type is supported (from FF), new business, and if the NR has been approved.

Reply:
I think you can remove line 416 since line 417 will cover it.

Where did you copy the "if approved" logic from? It may be complicated that this (ie, conditional approval). Not certain but asking if you've looked into it.

Reply:
The logic here is almost similar to the `showRegisterButton` on line 422 with the difference being instead of `isFirm`, it's `isSupportedEntity`. I do agree with you on the first point. I was just using the similar condition to the existing one (`showRegisterButton`).

Reply:
Please call out for some extra testing on acceptable NR states to incorporate from (and/or confirm what's done in Auth Web for IA NRs).

Otherwise, my concerns are satisfied here.

Reply:
I don't think so. I took a look and this should be OK.

Keep in mind that the component takes a prop that disables the button when the NR was approved but still in processing (5 minutes or so).